shipyard-controller/db/mongodb_sequence_execution_repo.go

```diff
@@ -348,6 +349,11 @@ func (mdbrepo *MongoDBSequenceExecutionRepo) getSearchOptions(filter models.Sequ
 	searchOptions = appendFilterAs(searchOptions, filter.Scope.Stage, "scope.stage")
 	searchOptions = appendFilterAs(searchOptions, filter.Scope.Service, "scope.service")
 	searchOptions = appendFilterAs(searchOptions, filter.CurrentTriggeredID, "status.currentTask.triggeredID")
+	if !filter.TriggeredAt.IsZero() {
```

Comment:
should we rather add a `From`/`To` field to the `SequenceExecutionFilter`? It might not be immediately clear that a `$lt` operator is applied if `TriggeredAt` is provided in the filter

Reply:
we actually want to filter always sequences which are less than the actual timestamp, I don't think `from`/`to` makes much sense here. If the `TriggeredAt` is not set, the behaviour of the filter stays unchanged and we do not need to deal with any incompatibilities. Also `from`/`to` will introduce extra logic, parameters and also slows the request when applied every time